packages/loot-core/src/shared/util.test.ts

```diff
@@ -10,12 +10,16 @@ describe('utility functions', () => {
     expect(looselyParseAmount('3.45')).toBe(3.45);
```

Comment:
There is a comment a few lines above this that should be updated for the new parsing.